EvolvSwiftSDK/EvolvSwiftSDK/Sources/Core/EvolvStoreImpl.swift

```diff
@@ -63,12 +66,18 @@ public class EvolvStoreImpl: EvolvStore {
                 }, receiveValue: { [weak self] (configuration, allocations) in
                     self?._evolvConfiguration = configuration
                     self?.evolvAllocations = allocations
+                    self?.evolvContext.contextChanged(key: "experiments.allocations", value: allocations, before: [])
```

Comment:
I don't quite follow this bit -- shouldn't we just be setting the experiment.allocations value on the context, then this would automatically fire?  experiments.allocations should be set in the remote context

Reply:
I store allocations in the Store object. I'm afraid you can't automatically fire events without knowing what exactly is changed in the context.